tests/utils/test_icon.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+import mock
+
+from gi.repository import GdkPixbuf
+from ubuntucleaner.utils.icon import get_from_name, DEFAULT_SIZE
+
+
+class TestIconModule(unittest.TestCase):
+
+    def test_get_from_name(self):
+        """
+        Calling with no parameters should return the pixbuf object through `load_icon()`
+        and default parameters.
+        """
+        m_pixbuf = mock.Mock(spec=GdkPixbuf.Pixbuf, get_height=lambda: DEFAULT_SIZE)
```

Comment:
note the `get_height()` is set so we don't reach this part https://github.com/gerardpuig/ubuntu-cleaner/blob/12b7277/ubuntucleaner/utils/icon.py#L45 this part as it would be tested in another dedicated test